injective/assetlist.json

```diff
@@ -285,6 +285,50 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/hava.png"
         }
       ]
+    },
+    {
+      "description": "A clan of 11y bad kids crafting chaos on the Cosmos eco. One bad memecoin to rule them all  $BADKID. Airdropped to Badkids NFT holders and $STARS stakers. It's so bad, your wallet's throwing a tantrum for it.",
+      "denom_units": [
+        {
+          "denom": "ibc/A0C5AD197FECAF6636F589071338DC7ECD6B0809CD3A5AB131EAAA5395E7E5E8",
+          "exponent": 0
+        },
+        {
+          "denom": "BADKID",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/A0C5AD197FECAF6636F589071338DC7ECD6B0809CD3A5AB131EAAA5395E7E5E8",
+      "name": "Badkid",
+      "display": "BADKID",
+      "symbol": "BADKID",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "factory/osmo10n8rv8npx870l69248hnp6djy6pll2yuzzn9x8/BADKID",
+            "channel_id": "channel-8"
+          },
+          "chain": {
+            "channel_id": "channel-122",
```

Comment:
I think you have these channels backwards (it's 122 on the osmosis side, not the injective side), which is further evidenced by the calculated ibc denoms not matching what you have here. I think you have the same issue with the other asset on the neutron assetlist